scripts/eval/eval.py

```diff
@@ -56,6 +56,15 @@ def load_model(model_cfg: DictConfig, tokenizer: PreTrainedTokenizerBase,
     return composer_model
 
 
+def log_analytics_details(mosaicml_logger: MosaicMLLogger, model_name: str):
+    mosaicml_logger.log_metrics({
+        'uses_llmfoundry': True,
```

Comment:
I actually don't think we need this directly - presence of any foundry metadata will be enough on the analytics side to know the run uses foundry